components/feature/media/src/main/java/mozilla/components/feature/media/ext/SessionState.kt

```diff
@@ -11,13 +11,21 @@ import mozilla.components.browser.state.state.SessionState
 import mozilla.components.concept.engine.mediasession.MediaSession
 import mozilla.components.feature.media.R
 
-internal fun SessionState?.getTitleOrUrl(context: Context): String = when {
+internal fun SessionState?.getTitleOrUrl(context: Context, title: String? = null): String = when {
     this == null -> context.getString(R.string.mozac_feature_media_notification_private_mode)
     content.private -> context.getString(R.string.mozac_feature_media_notification_private_mode)
     content.title.isNotEmpty() -> content.title
+    title != null -> title
```

Comment:
Shouldn't this be higher so that we prefer to use this over the page's title? 🤔

Reply:
yes you're right.  it should be higher.  I'll fix this.